dashboard/src/components/Map/Map.js

```diff
@@ -11,11 +11,47 @@ import {
 
 import features from "./features.json";
 import vulnerabilityData from "./vulnerability.csv";
+import { getCountries } from '../../api';
 
 const MapChart = () => {
   const [data, setData] = useState([]);
+  const [parameter, setParameter] = useState("");
+
+  const parameters = [
+    "humidity",
+    "pm1",
+    "pm10",
+    "pm25",
+    "pressure",
+    "temperature",
+    "um003",
+    "um005",
+    "um010",
+    "um025",
+    "um050",
+    "um100"
+  ];
 
   useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const response = await getCountries(); // Use the API function
+        const mappedData = response.results.map(result => ({
+          ...result,
+          measurements: result.measurements.reduce((acc, measurement) => ({
+            ...acc,
+            [measurement.parameter]: measurement.value
+          }), {})
+        }));
+        setData(mappedData);
+        console.log(mappedData); // Print response data to console
```

Comment:
Are we expected to have air quality data for all countries? The array in the console appears to be missing data for some countries.

Reply:
The data from the API isn't complete for every country.  One reason why I was messing with the other APIs.  We'll need to set a color code for "data unavailable" for missing values, I guess?

Reply:
Actually for that matter, I don't think it has every country.  So we need to think about that.  I'll keep looking for APIs that have additional data, as well

Reply:
Okay, I understand. Currently, we can assign a specific color to represent countries with missing data and focus on developing other app features. However, please continue your search for alternative APIs that can provide additional data to improve the application's coverage and availability of country information.

Regarding the detailed view, we require data for cities, and for the historical view, we need historical air quality data for cities. Do you think this API can fulfill these requirements?

Reply:
There are endpoints for both cities and countries, so I don't see that as a problem.  I'm not quite as confident about the historical data, but this endpoint:  https://docs.openaq.org/reference/measurements_get_v2_measurements_get takes From/To date parameters.  Assuming this PR is good to push to main, I will start working on additional API functions for city/country and historical queries.

Reply:
That's great that the API has endpoints for cities and countries, and the mentioned endpoint allows retrieving measurements based on specific date ranges. So, as you mentioned, you can work on implementing features to fetch data for cities and historical records to use in the views. If you come across better APIs, we can update the implementation accordingly.

Reply:
Excellent.  Corey, unless you have any additional thoughts on this PR, I'm going to go ahead and merge into main in a little bit.